docs/resources/application.md

```diff
@@ -116,6 +122,8 @@ resource "fusionauth_application" "Forum" {
 * `multi_factor_configuration` - (Optional)
     - `email_template_id` - (Optional) The Id of the email template that is used when notifying a user to complete a multi-factor authentication request.
     - `sms_template_id` - (Optional) The Id of the SMS template that is used when notifying a user to complete a multi-factor authentication request.
+    - `login_policy` - (Optional) When enabled and a user has one or more two-factor methods configured, the user will be required to complete a two-factor challenge during login. When disabled, even when a user has configured one or more two-factor methods, the user will not be required to complete a two-factor challenge during login. When required, the user will be required to complete a two-factor challenge during login.
```

Comment:
Could you list the acceptable values for these fields here?

Reply:
Fixed via aa03def! 👍